dataprofiler/profilers/profile_builder.py

```diff
@@ -365,6 +383,34 @@ def __init__(self, data, samples_per_update=None, min_true_samples=None,
         if isinstance(data, data_readers.text_data.TextData):
             raise TypeError("Cannot provide TextData object to Profiler")
 
+        # assign data labeler
+        use_data_labeler = True
+        data_labeler_dirpath = None
+        self.data_labeler = None
+        structured_options = None
+
+        if profiler_options and profiler_options.structured_options:
+            structured_options = profiler_options.structured_options
+
+        if structured_options and isinstance(
+                structured_options, StructuredOptions):
+            data_labeler_options = structured_options.data_labeler
+
+        if data_labeler_options and isinstance(
+                data_labeler_options, DataLabelerOptions):
+            use_data_labeler = data_labeler_options.is_enabled
```

Comment:
My two cents is that data_labeler should be added to the data labeler column options and that there is no need for all the function signatures to change at all. 

Also, some/all these if statements can be combined